### PR TITLE
Temporarily disable replication slave checking

### PR DESCRIPTION
**Pull request by mnagy**
As agreed with @bparees, temporarily disable some features of extended MySQL replication tests so tests pass and @php-coder can get his #6197 merged.

### CODE REVIEW

**Comment by mnagy** [comment]
[testonlyextended][extended:core(images: mysql)]

**Comment by bparees** [comment]
i thought you were just doing to disable a master-restart test.  this seems like more?

**Comment by mnagy** [comment]
Yeah, this disables verifying that the slave has replicated data from master. Unfortunately, only disabling the restarts still flakes for me.

**Comment by bparees** [comment]
so we don't even know that we have a working replication solution when no restarts happen?  ouch.

**Comment by bparees** [comment]
what's the behavior like when you manually go through the test steps?

**Comment by mnagy** [comment]
So, the replication I/O slave thread dies for 5.5 only. I refactored the tests a little bit and disabled the replication tests for 5.5.

**Comment by bparees** [comment]
lgtm.  will wait for outcome of test run.

**Comment by bparees** [comment]
[merge]

**Comment by soltysh** [comment]
Relates to #5352

**Comment by mnagy** [comment]
The fail is due to `pkg/generate/app/cmd` test timing out. On my local box the test takes around 80s. Definitely not related to this PR.

File: test/extended/images/mysql_replica.go [hunk 1]
```diff
@@ -111,6 +126,10 @@ func replicationTestFactory(oc *exutil.CLI, template string) func() {
 		g.By("after initial deployment")
 		master, _, _ := assertReplicationIsWorking("mysql-master-1", "mysql-slave-1", 1)
 
+		if tc.SkipReplication {
```
Comment (mfojtik) [inline test/extended/images/mysql_replica.go]: add log message here that something was skipped.

**Comment by mfojtik** [comment]
[merge]

**Comment by bparees** [comment]
[merge]

**Pull request merged**